app/src/main/java/com/android_academy/covid_19/repository/InfectionDataRepo.kt

```diff
@@ -35,9 +35,9 @@ class InfectionDataRepoImpl(
         // here in the future we need to call for a server only for delta,
         // but currently we call for a complete data from server
         // val lastInfectedLocation =  dao.getLastUpdatedInfectedLocation()
-        val fromServer = service.getInfectedLocationsMOH()
+        val fromServer = service.getInfectedLocations("covid-19",null, "carrier", "il")
```

Comment:
Please take out the strings to reusable public constants, or set the as default values in the function signature